config/default/manager_image_patch.yaml

```diff
@@ -8,5 +8,5 @@ spec:
     spec:
       containers:
       # Change the value of image field below to your controller image URL
-      - image: gcr.io/k8s-staging-cluster-api/capk-manager:dev
+      - image: localhost:49293/capk-manager-amd64:dev
```

Comment:
This change seems to be not intentional.

Reply:
You right, removed this change